Tevinthuku/Ocaml-flavored-politico
# **What does this PR do?**

**Description of Task to be completed?**



- [x] task one
- [x] task two

**How should this be manually tested?**
After cloning the repository to your local machine - check
[README.md](README.md) for instructions. Using
[Postman](https://www.getpostman.com/) open create a request with this
header
`Key Content-Type: application/json`

**Any background context you want to provide?**
An reason behind the implementation.

**What are the relevant pivotal tracker stories?**
[](https://www.pivotaltracker.com/n/projects/2241721/stories/story_id)

**Screenshots (if appropriate)**
